bookwyrm/models/book.py

```diff
@@ -375,15 +382,7 @@ def save(self, *args: Any, **kwargs: Any) -> None:
         # Create sort title by removing articles from title
         if self.sort_title in [None, ""]:
```

Comment:
We probably don't need to check this twice?

Reply:
haha oops -- the famous "wait, did I check that already" programming philosophy